WikiWatershed/model-my-watershed
# Show loading and error messages in Analyze view

To test:
 * Go to the analyze view
 * Check that there is a centered spinner that says Loading.
 * Check that results appear after a few seconds
 * Repeat, except right when spinner starts, switch the network to offline. (To switch the network to offline, in Developer Tools, click the smartphone icon in the upper left corner. Then back in the browser, there should be a network dropdown on the top.)
 * Check that an error message with icon shows. 
 * With network back to normal, check that you can go to the Analyze view, wait for results to show, and then go to the modeling view, and back. The results should show immediately.

I've only had time to fix the existing unit tests for the Analyze view. I also was unable to get the new animation for the Analyze view to work due to complicated CSS issues. I'll make issues for these in a minute.

Connects #80
